client/src/components/_common/Button/Button.jsx

```diff
@@ -140,6 +139,7 @@ Button.defaultProps = {
   iconNameAfter: '',
   type: 'secondary',
   size: '', // unless `type="link", defaults to `short` after `propTypes`
+  dataTestid: '',
```

Comment:
@wesleyboar Would `undefined` be a better default?

Reply:
I don't know, but it _is_ more semantically accurate **and** doesn't break the tests **and** renders the markup fine.

Changed (a6c81cd).